### DISCUSSION TITLE
How to access deployment id/url from CI (github actions)

### DISCUSSION THREAD

**Discussion by remjx** [vercel/vercel#6310] [Help]
In my GitHub Actions CI workflow, I need to access the deployment URL as an input to one of my build processes. 

I'm currently using Deploy Hooks to trigger redeploy after dependencies are published, but this only works for predefined branches.

It looks like there is no way to trigger a rebuild programmatically for a given deployment URL. Or generate a deployment ID and use a web hook to trigger the build. 

Is there any way to accomplish this with Vercel?

**Comment by leerob** [comment]
https://vercel.com/support/articles/using-private-dependencies-with-vercel

**Reply by remjx** [reply]
I am using this but the problem is that I need to know the Deployment URL as an input to some of my CI steps, and I need to wait/rebuild after my CI steps are complete
